cernopendata/modules/fixtures/data/records/cms-primary-datasets-HIpp-Run2011A.json

```diff
@@ -0,0 +1,737 @@
+      "links": [
+        {
+          "description": "Validated runs, full validation",
+          "recid": "14208"
+        },
+        {
+          "description": "Validated runs, muons only",
+          "recid": "14209"
+        }
+      ]
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "CMS collaboration",
+      "recid": "450"
+    },
+    "collections": [
+      "CMS-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "2.76TeV",
+      "type": "pp"
+    },
+    "date_created": [
+      "2011"
+    ],
+    "date_published": "2020",
+    "date_reprocessed": "2011",
+    "distribution": {
+      "formats": [
+        "root",
+        "reco"
+      ],
+      "number_events": 56168211,
+      "number_files": 1491,
+      "size": 6570934150548
+    },
+    "experiment": "CMS",
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "</p><p><strong>Data taking / HLT</strong><br/>The collision data were assigned to different RAW datasets using the following <a href=\"/record/1700\">HLT configuration</a>.</p><p><strong>HLT trigger paths</strong><br/>The possible HLT trigger paths in this dataset are:<br/><a href=\"/search?q=HLT_Jet20\">HLT_Jet20</a><br/><a href=\"/search?q=HLT_Jet40\">HLT_Jet40</a><br/><a href=\"/search?q=HLT_L1BscMinBiasORBptxPlusANDMinus\">HLT_L1BscMinBiasORBptxPlusANDMinus</a><br/><a href=\"/search?q=HLT_L1DoubleForJet32_EtaOpp\">HLT_L1DoubleForJet32_EtaOpp</a><br/><a href=\"/search?q=HLT_L1DoubleForJet8_EtaOpp\">HLT_L1DoubleForJet8_EtaOpp</a><br/><a href=\"/search?q=HLT_Mu0\">HLT_Mu0</a><br/><a href=\"/search?q=HLT_Photon10_CaloIdVL\">HLT_Photon10_CaloIdVL</a><br/><a href=\"/search?q=HLT_Photon15_CaloIdVL\">HLT_Photon15_CaloIdVL</a><br/><a href=\"/search?q=HLT_PixelTracks_Multiplicity50_Loose\">HLT_PixelTracks_Multiplicity50_Loose</a><br/><a href=\"/search?q=HLT_PixelTracks_Multiplicity60_Loose\">HLT_PixelTracks_Multiplicity60_Loose</a><br/><a href=\"/search?q=HLT_PixelTracks_Multiplicity70_Loose\">HLT_PixelTracks_Multiplicity70_Loose</a><br/><a href=\"/search?q=HLT_ZeroBiasPixel_SingleTrack\">HLT_ZeroBiasPixel_SingleTrack</a></p>"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "14016",
+    "relations": [
+      {
+        "title": "/AllPhysics2760/Run2011A-v1/RAW",
+        "type": "isChildOf"
+      }
+    ],
+    "run_period": [
+      "Run2011A"
+    ],
+    "system_details": {
+      "global_tag": "GR_R_44_V15::All",
+      "release": "CMSSW_4_4_7"
+    },
+    "title": "/AllPhysics2760/Run2011A-16Jul2011-v1/RECO",
+    "title_additional": "AllPhysics2760 primary dataset in RECO format from the 2.76 TeV p-p run of 2011 (/AllPhysics2760/Run2011A-16Jul2011-v1/RECO)",
```

Comment:
Perhaps we can use "proton-proton" instead of "p-p"? 

But the most important question is what are we going to do with the data files? I don't think we have them on EOSPUBLIC already? Shall we merge with an "on demand" flag?

Reply:
Proton-proton is better indeed 👍 .
According to my book-keeping (in https://github.com/cernopendata/data-curation/issues/73), the datasets were transferred. The only two missing are the RECO versions of already existing 2011 MinimumBias datasets, but I did not build the records for them.

Reply:
Ah, they are living under `data`, while I had looked under `hidata`. Where do we place them? E.g. `Nov2011_HI` would create a new top-level directory in `/eos/opendata/cms`...

Reply:
Ah, OK, that makes sense as they are pp data and they ended up there because of their naming in CMS system. They are part of Run2011A so they can go under there as the other Run2011A datasets.

Reply:
Run2011A looks good, but Nov2011_HI is living outside, e.g.:

```console
$ eos ls /eos/opendata/cms/upload/data/Nov2011_HI
AllPhysics2760
$ eos ls /eos/opendata/cms/upload/data/Run2011A
AllPhysics2760
MinBias0Tesla0
MinBias0Tesla1
MinBias0Tesla2
MuEG
MuOnia
```

Hence probably the need for a new top-level directory Nov2011_HI...

Reply:
OK, that makes sense as well. The three datasets which appear under ther three different directories under RECO there are different skims from the full dataset which resides under Run2011A. Let's keep this same logic if it does not hurt.

Reply:
I've started the data transfer and update this after merge...